website/docs/docs/collaborate/explore-multiple-projects.md

```diff
@@ -4,31 +4,47 @@ sidebar_label: "Explore multiple projects"
 
-<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-parent.png" width="70%" height="100" title="Cross-project lineage in a parent project"/>
+<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-parent.png" width="100%" height="100" title="View your cross-project lineage in a parent project and the other projects that reference it by clicking the 'Relationships' tab."/>
 
-When viewing a downstream (child) project that imports and refs public models from upstream (parent) projects, public models will show up in the lineage graph and display an icon on the graph edge that indicates what the relationship is to a model from another project. Hovering over this icon indicates the specific dbt Cloud project that produces that model. Double-clicking on a model from another project opens the resource-level lineage graph of the parent project, which is subject to your permissions.
+When viewing a downstream (child) project that imports and refs public models from upstream (parent) projects:
+- Public models will show up in the lineage graph and you can click on them to view the model details.
+- Clicking on a model opens a side panel containing general information about the model, such as the specific dbt Cloud project that produces that model, description, package, and more.
+- Double-clicking on a model from another project opens the resource-level lineage graph of the parent project, which is subject to your permissions.
 
-
-<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-lineage-child.png" width="85%" height="100" title="Cross-project lineage in a child project"/>
+<Lightbox src="/img/docs/collaborate/dbt-explorer/cross-project-child.png" width="100%" height="100" title="View a downstream 'child' project that importants and refs public models from the upstream 'parent' project."/>
 
 ## Explore the project-level lineage graph
 
 For cross-project collaboration, you can interact with the DAG in all the same ways as described in [Explore your project's lineage](/docs/collaborate/explore-projects#project-lineage) but you can also interact with it at the project level and view the details. 
 
-To get a list view of all the projects, select the account name at the top of the **Explore** page near the navigation bar. This view includes a public model list, project list, and a search bar for project searches. You can also view the project-level lineage graph by clicking the Lineage view icon in the page's upper right corner.
+To get a list view of all the projects in your account:
+- Navigate to the top left section of the **Explore** page, near the navigation bar.
+- Hover over the project name and select the account name.
+- This will return a project view page, allowing you to see all the projects in the account. It returns a public model list, project list, and a search bar for project searches.
```

Comment:
not sure what "this" is referring to here in this bullet. sugg improving for clarity

Reply:
changed it to:

< - Hover over the project name and select the account name to return a project view page, allowing you to see all the projects in the account.